frontend/src/pages/PostDetail/PostDetailPage.jsx

```diff
@@ -20,19 +20,21 @@ function formatDate(date) {
 /*
   Function used to recursively retrieve and display child comments
 */
-function nestComments(commentList, setModal) {
-  if (commentList) {
+function nestComments(commentList, setModal, setParentID) {
```

Comment:
This kind of function declaration isn't very consistent with the rest of the code. From what I noticed, most of them use arrows.

Reply:
I have just kept the existing function declaration and updated the parameters. Should I leave it how it was before we came, or update it to make it more consistent?

Reply:
Oh... If you're going to make changes which are mentioned in my other comment then I'd like it changed, if not then leaving it is fine.

Reply:
Alright, I'll update the eslint rules and convert this to an arrow function.